pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
I don't think you have the correct behavior here (or at least the tests don't) for a typical usage of

``downcast='int'``, meaning I want to downcast not to what 'int' does, but to the smallest int possible (e.g you need to use the original compact_int code). you need to document this as well. (same usage for float/unsigned). This provides flexbility while getting the desired behavior.